JS/lib/cards.js

```diff
@@ -0,0 +1,142 @@
+//   <section id="question-card0">
+//   <div class="question-card__total">
+//     <h2>QUESTION X</h2>
+
+//     <p>Can you die without perishing ?</p>
+
+//     <button class="question-card__answer">Show Answer</button>
+
+//     <i class="fas fa-bookmark question-card__bookmark-non"></i>
+//     <p class="answer">
+//       When your body leaves your soul, your soul will be forever
+//     </p>
+//   </div>
+// </section>
```

Comment:
Since you merging this branch into your `production` branch `main` you should delete this commented code.